### DISCUSSION TITLE
two points given((x1,y1),(x2,y2)),how to draw a arc with given radius

### DISCUSSION THREAD

**Discussion by luzhoumingyue** [gdsfactory/gdsfactory#1469] [General]
Hi sir,if i defined two points,just as ((x1,y1),(x2,y2)),and defined radius,how to draw a arc?just like add_polygon,but add_polygon use straight,not arc,thank u so much~

**Comment by simbilod** [comment]
Look into Paths https://gdsfactory.github.io/gdsfactory/notebooks/03_waveguides_paths_crossections.html

**Reply by jan-david-fischbach** [reply]
Updated link is https://gdsfactory.github.io/gdsfactory/notebooks/03_Path_CrossSection.html
